CPAC/utils/datasource.py

```diff
@@ -409,8 +409,8 @@ def ingress_func_metadata(wf, cfg, rpool, sub_dict, subject_id,
 
             fmap_rp_list.append(key)
 
-            if key == "diff_phase" or key == "diff_mag_one" or \
-                            key == "diff_mag_two":
```

Comment:
`"diff_mag_two"` is not relevant here anymore?

Reply:
@amygutierrez resolved the `bytes` TO-DO (I just checked it off).
I'll update the changelog shortly - thanks for reminder!

Reply:
Not with the way we implement this type of distcor. But we want to revisit, especially regarding when we apply the unwarp (probably should apply before co-reg).

Reply:
Second diff mag can be used in some cases.